Lib9c/Action/ClaimItems.cs

```diff
@@ -70,62 +71,91 @@ public override IAccount Execute(IActionContext context)
             var random = context.GetRandom();
             var itemSheet = states.GetSheets(containItemSheet: true).GetItemSheet();
 
-            foreach (var (avatarAddress, fungibleAssetValues) in ClaimData)
+            foreach (var (recipientAddress, fungibleAssetValues) in ClaimData)
             {
-                var inventoryAddress = avatarAddress.Derive(LegacyInventoryKey);
-                var inventory = states.GetInventory(inventoryAddress)
-                                ?? throw new FailedLoadStateException(
-                                    ActionTypeText,
-                                    GetSignerAndOtherAddressesHex(context, inventoryAddress),
-                                    typeof(Inventory),
-                                    inventoryAddress);
+                var inventoryAddress = recipientAddress.Derive(LegacyInventoryKey);
+                Inventory inventory = null;
+                bool inventoryExist = false;
+                try
+                {
+                    inventory = states.GetInventory(inventoryAddress);
+                    inventoryExist = true;
+                }
+                catch (Exception)
```

Comment:
I think that catching `Exception` instead of specific exception for inventory missing would make harder to debug in some circumstances.